<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>duplicates</class>
 <widget class="QWidget" name="duplicates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1558</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duplikaty</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>681</width>
     <height>101</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń wybrany duplikat</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>691</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>871</width>
     <height>791</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>0</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duplikaty</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pasek postępu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tablica Haszów</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
